template/lambda-sqs-worker-cdk/src/app.ts

```diff
@@ -1,8 +1,10 @@
 import createLogger from '@seek/logger';
 import type { SQSEvent, SQSHandler } from 'aws-lambda';
+import { config } from 'infra/config';
```

Comment:
Thanks for the contribution @GeordieEK!

@samchungy I know you have opinions on how config is structured for CDK applications. Should we share config between the infra stack & runtime app?

Reply:
Hmmm I'm trying to think if there's any issues with them being separate. It does feel wrong to intermix them somehow.

Should they be separate? 🤔

Reply:
I think we want to keep them separate. If there are some environment variables we want to set just for the CDK infra deployment, we would always need to set them even if we were simply running the application.

I think we should keep our app config separate from our infra config. Especially given the uptake in monorepos

Reply:
I've separated infra and runtime configs but can't figure out how to fix linting errors

Reply:
Ah good attempt! The template can be a little tough to work with tbh. I'll have a quick looksie tomorrow morning

Reply:
I tend to run the test:template command to essentially scaffold the template and then edit it within there

Reply:
Yeah! I tried the test:template command, very nice. It makes clear what is causing it but I'm not sure why it's complaining for the cdk template but not the others that appear to me to have similar structure. Appreciate it :)

Reply:
I've pushed up some changes. If you jump to ../tmp-skuba/tmp-lambda-sqs-worker-cdk you should be able to see linting errors directly within your IDE.

Reply:
Ahh, that's the trick I was missing. Thanks so much!